neo/rawio/maxwellrawio.py

```diff
@@ -185,7 +185,7 @@ def _get_analogsignal_chunk(self, block_index, seg_index, i_start, i_stop,
                 # to be indexed out of order
                 sorted_channel_indexes = np.sort(channel_indexes)
                 resorted_indexes = np.array(
-                    [list(channel_indexes).index(ch) for ch in sorted_channel_indexes])
+                    [list(sorted_channel_indexes).index(ch) for ch in channel_indexes])
```

Comment:
@hornauerp I think there is a better implementation.

Can you try if this gives the same result:
```
order_f = np.argsort(channel_indexes)
sorted_channel_idxeses = channel_indexes[order_f]
# use argsort again on order_f to obtain resorted_indexes
resorted_indexes = np.argsort(order_f)
```

Reply:
Yes, same outcome, but slightly faster. (17 vs 50 us)

I just wanted to stick to the original code as much as possible to facilitate the review process, but yours is definitely the more optimal implementation.